src/test/scala/com/mozilla/telemetry/streaming/TestEventsToAmplitude.scala

```diff
@@ -92,6 +92,7 @@ class TestEventsToAmplitude extends FlatSpec with Matchers with BeforeAndAfterAl
   override def afterEach {
     verify(expectedTotalMsgs,
       requestMadeFor(new ValueMatcher[Request] {
+        // scalastyle:off methodName
```

Comment:
It seems we should instead update the regex to allow this kind of method name, with both a preceding and following back-tick.

Reply:
+1